lab-bundle-spec.md

```diff
@@ -1,18 +1,26 @@
 # Qwiklabs Lab Bundle Specification
 
-**Version 1**
+**Version 2**
 
 > This is a DRAFT document. We welcome feedback as this format evolves.
 
 Previously (in b6086b8f824aa398c1f4413b92351a4956e744cd), the robust example had some cool ideas for how deployment manager and activity tracking should look in the future. None of it is implemented yet but the ideas may be useful in the future.
 
+## Changelog
+
+The primary changes made from v1 to v2 are:
+- Renaming the `fleet` attribute on the `gcp_project` resource type to `variant`, and allowing for `variant` to be specified on other resource types.
+- Replacing the `dm_template` attribute on the `gcp_project` resource type with a `startup_script` and `cleanup_script`.
+- Allowing `custom_properties` to include a `reference` (rather than `value`) that can be filled in from running resources.
+- Adding an `ssh_key_user` attribute on the `gcp_project` resource type.
+
 ## `qwiklabs.yaml` Structure
 
 Here's a sample `qwiklabs.yaml` file, with all nested details removed to make it easier to see the general file structure.
 
 ```yml
 entity_type: Lab
-schema_version: 1
+schema_version: 2
```

Comment:
Note that @jgan is working on creating a PR on top of this PR to:
- Include AWS considerations
- Rework `legacy_display_options`

Reply:
In practice, does the v2 bundle spec need to support `legacy_display_options`?

Reply:
No it does not. Josh's PR will be removing it.